README.md

```diff
@@ -62,8 +62,10 @@ Move to [awesome-docsify](https://github.com/docsifyjs/awesome-docsify#showcase)
 
 | Project                                          | Description                              |
 | ------------------------------------------------ | ---------------------------------------- |
-| [docute](https://github.com/egoist/docute)       | 📜 Effortlessly documentation done right |
```

Comment:
And the docute website has even be replaced with a scam website that tries to hack people who don't know any better!

https://github.com/egoist/docute/issues/317